superset/connectors/sqla/models.py

```diff
@@ -99,21 +104,20 @@ class TableColumn(Model, BaseColumn):
         s for s in export_fields if s not in ('table_id',)]
     export_parent = 'table'
 
-    @property
-    def sqla_col(self):
-        name = self.column_name
+    def sqla_col(self, db_engine_spec, label=None):
+        col_label = get_column_label(db_engine_spec, self.column_name, label)
```

Comment:
Just had a moment thinking about whether `verbose_name` should ever make its way in here and concluded no at least for now.